qcelemental/molutil/test_align.py

```diff
@@ -442,3 +442,77 @@ def test_hessian_align():
 
     p2chess = mill.align_hessian(p4_hooh_hess_native)
     assert compare_values(c4_hooh_hess, p2chess, atol=1.e-4)
+
+def test_vector_gradient_align():
```

Comment:
because `atoms_map=False` invokes the reordering machinery, need `@using_networkx` for this test like [here](https://github.com/MolSSI/QCElemental/pull/78/files#diff-c80b22cde9934b3725bea9e0aa7dcf1cR206).